job-dsl-plugin/build.gradle

```diff
@@ -95,9 +95,9 @@ dependencies {
     }
     optionalJenkinsPlugins 'org.jenkins-ci.plugins:config-file-provider:2.15.4'
     optionalJenkinsPlugins 'org.jenkinsci.plugins:managed-scripts:1.3'
-    optionalJenkinsPlugins 'io.jenkins:configuration-as-code:1.13'
-    jenkinsTest 'io.jenkins:configuration-as-code:1.13'
-    jenkinsTest 'io.jenkins:configuration-as-code:1.13:tests'
+    optionalJenkinsPlugins 'io.jenkins:configuration-as-code:1.15'
```

Comment:
As of today there is already [v1.20](https://github.com/jenkinsci/configuration-as-code-plugin/releases/tag/configuration-as-code-1.20).